mlflow/pyspark/ml/__init__.py

```diff
@@ -530,7 +530,7 @@ def is_metric_value_loggable(metric_value):
         as an MLflow metric.
         """
         return isinstance(metric_value, (int, float, np.number)) and not isinstance(
-            metric_value, bool
+            metric_value, (bool, np.bool_)
```

Comment:
Add `np.bool_` check.

Reply:
My original intention is to filter out bool values (including python bool type and numpy bool type). But original code I wrote `np.bool` but actually it should be `np.bool_` according to my original aim.

Reply:
Create separate PR https://github.com/mlflow/mlflow/pull/5462